Week-12/todo-list/src/Components/Todo.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import '../Style/Todo.css';
+import ListItem from "./ListItem";
+
+const Todo = () => {
+
+    const [list, setList] = useState(['Coding', 'Eating', 'Sleeping', 'Debugging']);
+    const addItem = () => {
+        setList([...list, '']);
+    }
+    const removeItem = (index) => {
+        let newList = list;
```

Comment:
this copies by reference and can cause bugs. Also, newList should be a const.

Reply:
updated this as well, made it const and not copying by reference